R/pkg/R/context.R

```diff
@@ -345,7 +351,8 @@ spark.getSparkFilesRootDirectory <- function() {
 #'}
 #' @note spark.getSparkFiles since 2.1.0
 spark.getSparkFiles <- function(fileName) {
-  callJStatic("org.apache.spark.SparkFiles", "get", as.character(fileName))
+  rootDir <- spark.getSparkFilesRootDirectory()
+  paste0(rootDir, "/", as.character(fileName))
```

Comment:
Do you suggest to write as follow?
```
if (Sys.getenv("SPARKR_IS_RUNNING_ON_WORKER") == "") {
    callJStatic("org.apache.spark.SparkFiles", "get", as.character(fileName))
  } else {
    file.path(spark.getSparkFilesRootDirectory(), as.character(fileName))
  }
```
It will check ```Sys.getenv("SPARKR_IS_RUNNING_ON_WORKER")``` twice for the case of running on worker, since we always make this check in ```spark.getSparkFilesRootDirectory```. So I'm more prefer the current style.